Automattic/jetpack
# About Page: remove submenu and add link to page in footer

Fixes #12315

#### Changes proposed in this Pull Request:

You can now access the About Page from 4 different locations:

- The Automattic link at the bottom of the Jetpack dashboard pages.
- The About link at the bottom of the Jetpack dashboard pages.
- The Automattic logo on any Jetpack non-react admin pages (e.g. Settings > Sharing)
- The about link at the bottom of any Jetpack non-react admin pages (e.g. Settings > Sharing)

#### Testing instructions:

* Go to any of the pages I mentioned above?
* Do you see the links?
* Can you click on them, does the page open in the same tab?

**Note**: 
- If your site is connected to wpcom or in dev mode, the About link will lead to the in-plugin About page.
- If you are not connected yet, the About link will lead to Jetpack.com.

#### Proposed changelog entry for your changes:

* None
